stdlib/public/core/Codable.swift.gyb

```diff
@@ -1101,7 +1101,7 @@ public struct CodingUserInfoKey : RawRepresentable, Equatable, Hashable {
 //===----------------------------------------------------------------------===//
 
 /// An error that occurs during the encoding of a value.
-@_fixed_layout // FIXME(sil-serialize-all)
+@_frozen // FIXME(sil-serialize-all)
```

Comment:
This is a surprise to me; why should encoding and decoding errors be frozen? I'd expect this to grow over time, like most error types.

Reply:
They'll definitely get un-frozen. This commit did not remove any `frozen`s that were currently there.

Reply:
... it did in stdlib/public/Platform/POSIXError.swift

Reply:
…oops. Okay, I'll be consistent.